cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -273,6 +277,7 @@ def test_course_info_updates_import_export(self):
             on_disk = loads(grading_policy.read())
             self.assertEqual(on_disk, course_updates.items)
 
+    @skip("OldMongo Deprecation")
```

Comment:
Why this tests are failing if we haven't changed code, is it because of the test utils changes?

Reply:
We may need to somehow overcome this skips, before we can merge this PR

Reply:
I'm going to fix it in next commit in `2nd Complexity level tests` task